elliottlib/util.py

```diff
@@ -247,3 +247,12 @@ def override_product_version(pv, branch):
             return 'OSE-{}-RHEL-{}'.format(rb[1], rb[3])
         return 'RHEL-{}-OSE-{}'.format(rb[3], rb[1])
     return pv
+
+
+def get_release_version(pv):
+    """ there are two kind of format of product_version: OSE-4.1-RHEL-8 RHEL-7-OSE-4.1 """
+    res = pv.split("OSE-", 1)[1]
```

Comment:
this... works, but just seems really clunky. how about:
```suggestion
    return re.search(r'OSE-(\d+\.\d+)', pv).groups()[0]
```

Reply:
Yeah, regex is better